org.OpenT2T.Sample.SuperPopular.Thermostat/Wink Thermostat/v2/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "opent2t-translator-wink-thermostat",
+  "version": "1.0.0",
+  "description": "Wink Thermostat translator",
+  "author": "OpenT2T",
+  "license": "MIT",
+  "scripts": {
+    "test": "ava"
+  },
+      "ava": {
+      "files": [
+        "tests/*.js",
+        "!**/deviceconfig.js"
+      ]
+    },
+  "dependencies": {
+    "delay": "^1.3.1",
+    "opent2t": "*",
+    "opent2t-onboarding-winkhub": "*",
+    "opent2t-translator-helper-wink": "1.0.0",
+    "q": "1.4.1"
+  },
+  "devDependencies": {
+    "ava": "^0.15.2"
+  },
+  "opent2t": {
+    "interfaces": {
+      "org.OpenT2T.Sample.SuperPopular.Thermostat": {
+        "description": "OpenT2T SuperPopular Thermostat interface"
+      }
+    },
+    "translators": {
+      "TranslatorA": {
+        "description": "OpenT2T Wink Thermostat translator",
+        "interfaces": [
+          "./ThermostatInterface"
```

Comment:
>ThermostatInterface [](start = 13, length = 19)

This must match the interface module name on line 28, whatever you change it to.